docs/content/usage/use-cases/network-breach.md

```diff
@@ -8,44 +8,40 @@ weight: 3
+internet-facing DB servers, a [cryptomining operation that attacks WordPress sites](https://www.guardicore.com/2018/06/operation-prowli-traffic-manipulation-cryptocurrency-mining-2/) or any other malicious campaign – attackers are now trying to go deeper into your network.
 
-Infection Monkey will help you assess the impact of internal network breach, by trying to propagate within it
- using service vulnerabilities, brute-forcing and other safe attack methods.
+Infection Monkey will help you assess the impact of a future breach by attempting to propagate within your internal network using service vulnerabilities, brute-forcing and other safe exploiters.
 
 ## Configuration
 
-- **Exploits -> Exploits** You can review the exploits Infection Monkey will be using. By default all 
+- **Exploits -> Exploits** Here you can review the exploits the Infection Monkey will be using. By default all 
 safe exploiters are selected.
-- **Exploits -> Credentials** This configuration value will be used for brute-forcing. We use most popular passwords
- and usernames, but feel free to adjust it according to your native language and other factors. Keep in mind that long 
- lists means longer scanning times.
+- **Exploits -> Credentials** This configuration value will be used for brute-forcing. The Infection Monkey uses the most popular default passwords and usernames, but feel free to adjust it according to the default passwords common in your network. Keep in mind a longer list means longer scanning times.
 - **Network -> Scope** Make sure to properly configure the scope of the scan. You can select **Local network scan**
- and allow Monkey to propagate until maximum **Scan depth**(hop count) is reached or you can fine tune it by providing 
- specific network ranges in **Scan target list**. Scanning local network is more realistic, but providing specific 
- targets will make scanning process substantially faster.
-- **(Optional) Internal -> Network -> TCP scanner** You can add custom ports your organization is using.
-- **(Optional) Monkey -> Post Breach Actions** If you only want to test propagation in the network, you can turn off 
-all post breach actions. These actions simulate attacker's behaviour after getting access to a new system, but in no
- way helps to exploit new machines.
+ and allow Monkey to propagate until maximum **Scan depth**(hop count) is reached, or you can fine tune it by providing 
+ specific network ranges in **Scan target list**. Scanning a local network is more realistic, but providing specific 
+ targets will make the scanning process substantially faster.
+- **(Optional) Internal -> Network -> TCP scanner** Here you can add custom ports your organization is using.
+- **(Optional) Monkey -> Post-Breach Actions** If you only want to test propagation in the network, you can turn off 
+all post-breach actions. These actions simulate an attacker's behaviour after getting access to a new system but in no
```

Comment:
Are we using British or American spellings?

Reply:
Good catch — American. I changed to 'behavior'.